Hunting Queries/SecurityEvent/Least_Common_Parent_Child_Process.yaml

```diff
@@ -29,4 +34,23 @@ query: |
   | extend ParentChildPair = strcat(ParentProc , ' > ', Proc)
   | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), TimesSeen = count(), HostCount = dcount(Computer), Hosts = makeset(Computer), UserCount = dcount(SubjectUserName), Users = makeset(SubjectUserName) by ParentChildPair
   | where TimesSeen < Sensitivity
+  | extend timestamp = StartTimeUtc),
+  (WindowsEvent
+  | where TimeGenerated between(lookback..endtime)
+  | where EventID == 4688 
+  | extend ParentProcessName = tostring(EventData.ParentProcessName)
+  | where isnotnull(ParentProcessName)
+  | extend NewProcessName = tostring(EventData.NewProcessName)
+  | extend ProcArray = split(NewProcessName, '\\'), ParentProcArray = split(ParentProcessName, '\\')
```

Comment:
Well, the original is so dump that I think you should fix it. What they do is just split(...)[-1] to get the filename.... and in 7 lines.